usaspending_api/references/v2/views/filter_tree/tas.py

```diff
@@ -17,9 +17,17 @@ class TASViewSet(APIView):
 
     def _parse_and_validate(self, request):
 
-        data = {"depth": request.get("depth") or 0}
+        data = {"depth": request.get("depth") or 0, "filter": request.get("filter") or None}
```

Comment:
```suggestion
        data = {"depth": request.get("depth") or 0, "filter": request.get("filter")}
```

Reply:
Is it necessary to map these from request to data? TinyShield should handle what this is performed on the request